rsapi/commands.go

```diff
@@ -104,6 +105,12 @@ func (a *API) ParseCommand(cmd, hrefPrefix string, values ActionCommands) (*Pars
 					name, value)
 			}
 			*coerced = append(*coerced, APIParams{name: val})
+		case "*time.Time":
+			val, err := time.Parse(time.RFC3339, value)
```

Comment:
Looks like only CA and SS APIs accept `time.Time` attribute type and I believe they both will be okay with RFC3339. The CM15 API seems to use the `string` type for parameters such as `start_date` in audit entries query.